src/Cli/func/Helpers/PythonHelpers.cs

```diff
@@ -10,6 +10,14 @@
 
 namespace Azure.Functions.Cli.Helpers
 {
+    public enum PythonDependencyManager
```

Comment:
In .NET maming something as a manager often refers to classes that perform management operations. Could we use something like `PythonPackageTool`, `PythonDependencyTool` or `PythonPackageSystem` instead?

Reply:
We would also update any reference to `dependencyManager` to something that makes sense like `packageTool`